### ISSUE TITLE
Feature Request: Show current location - friends

### ISSUE THREAD

**Issue by ukmerlin** [jaszhix/NoMansConnect#181]
If we pick to show friends, would it be possible to highlight the current location?
Whether they are currently online or not is probably a problem but you could store 
the last known / registered location.

**Comment by jaszhix** [comment]
Yes, it would be possible to show their last known location/online status (of NMC app).
